defaults/main.yml

```diff
@@ -1,14 +1,23 @@
 ---
 varnish_package_name: "varnish"
 varnish_version: "4.1"
+# Varnish 5+ is currently only supported on Debian and Ubuntu.
+# varnish_version: "5.1"
 
 varnish_use_default_vcl: true
 varnish_default_vcl_template_path: default.vcl.j2
 
 varnish_default_backend_host: "127.0.0.1"
 varnish_default_backend_port: "8080"
+# If backend server expect PROXY protocol connexion:
+# varnish_default_backend_proxy_proto: 1
+
+varnish_listen_ports:
+  - port: 80
+  - port: 6081
+    # If Varnish must expect a PROXY protocol connexion:
+    proxy_proto: yes
```

Comment:
Yes; I'm working on the first two (really quick/simple wins there!), then will work on the 5.x support next—see related: https://github.com/geerlingguy/ansible-role-varnish/issues/58

I'd like to get this done soon since I want to support Debian 9/Stretch.